### DISCUSSION TITLE
Why don't test stable?

### DISCUSSION THREAD

**Discussion by ToBinio** [frewsxcv/rust-crates-index#141] [Q&A] [answered]
Just out of curiosity what is the reason that the test run on nighty, beta but not stable?

is it simple because when nighty and beta run stable will as well? 

couldn't then the same be said about beta? when nightly works beta will as well

**Comment by Byron** [comment, marked as answer]
I actually checked when adding `windows` runners but couldn't find anything interesting in the respective commit messages.

With that said, I believe that `lib.rs` is built with `nightly`, which is why the maintainer was particularly interested in having this tested.
Using that as an argument, then we should probably add `stable` just for the sake of knowing it works there, not just assuming it by merit that `nightly` should never be less compatible than stable, nor should `beta`.

If all that is true, and the environment should be a concern, it should be possible to only test on `stable` and half CI time again, after it was doubled by adding `windows` runners.

My preference here is to only test `stable` and wait for issues to show up that indicate reasons for testing non-stable releases of Rust.

**Reply by Byron** [reply]
And I have made the change, let's see how that alters timings especially when caches finally kick in.
